src/UI/docu/ux-guide-repository-objects-properties-and-actions.md

```diff
@@ -0,0 +1,364 @@
+interface to help display or hide information of an object outside of the item
+itself. For example, many file managers can display additional information and
+actions in a sidebar. Maybe the slate can be filled with new tools and features
+to aid in analyzing, comparing, and managing repository items.
+
+We also might want to more clearly and consistently decide on how a property is
+displayed - e.g. as a key text & value text, key text & icon, icon & value text,
+just the value text, or just as an icon. For example in "Status: Offline", the
+word "Status" seems to be redundant, as the word "Offline" alone communicates
+the same amount of information without the leading key.
+
+### User adjusted views
+
+A user with the intent to manage multiple objects would like to use filters,
+display properties, and sorting to shape the view to match their current focus.
+Currently, in the repository, there is no way for the user to (temporarily)
+choose a specific set of of properties to be displayed, while hiding others. We
+might want to consider adding a table view, where managing users can quickly
+show and hide property columns of a container, and can benefit from all the
+filter and sorting options they know from other table views.
+
+To declutter the content area, we might want to consider utilizing the slate for
+some of the filter and sorting tools. This, on the one side, keeps them in view
+while scrolling, and on the other side, gives an option to hide them for users
+who don't wish to use them.
+
+### Curated views
+
+Very often in ILIAS, we have higher user roles managing content for lower user
+roles. As part of this, the higher user roles craft landing pages with carefully
+selected repository items on them, combined with information added through the
+page editor. If the managing user would have more control over which properties
+are shown, they could build very focused views around the anticipated user
+intent.
+
+For example, a landing page for new users offering the only three mandatory
+beginner courses could be completely reduced to a UI Card with just a thumbnail
+and the course title. A catalogue of all available courses, on the other hand,
+could use the UI Presentation Table with focus on event dates to assist in
+scheduling the learner's upcoming months.
+
+We have seen managing users building such views manually with the page editor,
+faking a deck of cards that only contains the desired information and
+highlights. While this is a valid workaround, these fake items do not update
+when the object they represent is updated. Adding, updating, and removing many
+such items can turn into a laborious process, and is prone to user error.
+
+When concept designers and developers build views around a specific focus, they
+are creating a curated view as well. For example the badge, certificate, and
+contacts pages utilize a deck of cards with a carefully chosen selection of
+properties and actions.
+
+## Next steps
+
```

Comment:
Hi @atoedt,

what do you mean by "Project background and meeting history"? Thanks for the other input, though, will look through it with @catenglaender.

Best regards!

Reply:
i.e. line 93 ff. or line 325 ff. and similar
This is helpful to get a deeper understanding how this was arrived at. Howver if split up into different documents it would be more concise

Reply:
Thanks for the suggestion, but we decided to leave the structure of the document as is. Different approaches can be used to present similar information all the time, this is the want that we have decided to use.